### ISSUE TITLE
NPC: Hertical monk of brigh and Kurobozu need black apoxia trait on steal breath

### ISSUE THREAD

**Issue by ThingsCouldGetDicey** [foundryvtt/pf2e#8317]
https://2e.aonprd.com/Monsters.aspx?ID=1215

it was hard to follow quickly when you are running the game but it would be appreciated if the black apoxia trait was on steal breath. for both in the ap outlaws of alkenstar book 2 chapter 3 in the cradle of quartz chapter and for the original monster the korubozu. 

it would help understand whats using that disease. 

thank you.

**Comment by stwlam** [comment]
"Steal breath" is not a trait.

**Comment by ThingsCouldGetDicey** [comment]
> "Black Apoxia" is not a trait.

if you see the indication on the other strike it has sagebane. which isnt a "trait" but it helps know that its linked to the strike. 

id like the same tip info on steal breath for black apoxia 
![image](https://github.com/foundryvtt/pf2e/assets/62528192/6962e655-5abb-4427-856e-a3c12a7b3ee5)
![image](https://github.com/foundryvtt/pf2e/assets/62528192/bf5b648a-249d-420c-9aee-4d4514938b83)
